src/app/admin/subjects/subject-confirm/subject-confirm.component.html

```diff
@@ -0,0 +1,8 @@
+<h1 mat-dialog-title>Підтвердження</h1>
+<div mat-dialog-content>
+    <p>{{confirm}}</p>
+</div>
+<div mat-dialog-actions>
+    <button mat-button color="primary" (click)="onCensle()">Скасувати</button>
```

Comment:
Looks like there is a mistake here `onCensle()`

Reply:
Corrected